src/main/java/com/crud/tasks/mapper/TaskMapper.java

```diff
@@ -0,0 +1,31 @@
+package com.crud.tasks.mapper;
+
+import com.crud.tasks.domain.Task;
+import com.crud.tasks.domain.TaskDto;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class TaskMapper {
+    public Task mapToTask(final TaskDto taskDto) {
+        return new Task(
+                taskDto.getId(),
+                taskDto.getTitle(),
+                taskDto.getContent());
+    }
+
+    public TaskDto mapToTaskDto(final Task task) {
+        return new TaskDto(
+                task.getId(),
+                task.getTitle(),
+                task.getContent());
+    }
+
+    public List<TaskDto> mapToTaskDtoList(final List<Task> taskList) {
+        return taskList.stream()
+                .map(t -> new TaskDto(t.getId(), t.getTitle(), t.getContent()))
```

Comment:
Zauważ, że możesz tutaj użyć `this::mapToTaskDto`. Zaleta tego rozwiązania jest taka, że jeśli dodasz nowe pole do `Task` i/lub `TaskDto`, to będziesz miała jedno miejsce miej do przerabiania :)